app/src/main/kotlin/com/wire/android/ui/home/conversations/DownloadedAssetDialog.kt

```diff
@@ -0,0 +1,62 @@
+package com.wire.android.ui.home.conversations
+
+import androidx.activity.compose.rememberLauncherForActivityResult
+import androidx.activity.result.contract.ActivityResultContracts
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import com.wire.android.R
+import com.wire.android.ui.common.WireDialog
+import com.wire.android.ui.common.WireDialogButtonProperties
+import com.wire.android.ui.common.WireDialogButtonType
+import com.wire.android.util.permission.WriteToExternalStorageRequestFlow
+
+@Composable
+fun DownloadedAssetDialog(
+    conversationViewState: ConversationViewState,
```

Comment:
no need to pass ConversationViewState you can pass downloadedAssetDialogState instead

Reply:
Thanks. I passed the `DownloadedAssetDialogVisibilityState` instead